### PR TITLE
[Bulk| Validations: Fix missing alt text issues 4

### PR DESCRIPTION
**Pull request by v-lkasturi**
Added alternate texts for screenshots.

### CODE REVIEW

File: iis/media/transform-manager/encrypting-on-demand-smooth-streams.md [hunk 1]
```diff
@@ -63,29 +63,29 @@ You can also add the **PlayReady Protection** task to other built-in job templat
 To configure job templates, do the following:
 
 1. In IIS Manager, in the **Connections** pane, click **Job Templates**.  
-   ![](encrypting-on-demand-smooth-streams/_static/image1.png)
+   ![Screenshot shows Connections pane to select Job Templates.](encrypting-on-demand-smooth-streams/_static/image1.png)
```
Comment (RedZephyr13) [inline iis/media/transform-manager/encrypting-on-demand-smooth-streams.md:66]: How about "Screenshot that shows the Connections pane. Job Templates is selected under the Transform Manager node."

File: iis/media/transform-manager/encrypting-on-demand-smooth-streams.md [hunk 2]
```diff
@@ -63,29 +63,29 @@ You can also add the **PlayReady Protection** task to other built-in job templat
...
 
    | **Smooth Streams to encrypted Smooth Streams** | **Video files to H.264 Smooth Streams and Apple HTTP Live Streams** |
    | --- | --- |
-   | ![](encrypting-on-demand-smooth-streams/_static/image5.png) | ![](encrypting-on-demand-smooth-streams/_static/image6.png) |
+   | ![Screenshot shows a list of job templates with name and status.](encrypting-on-demand-smooth-streams/_static/image5.png) | ![Screenshot shows a list of job templates to select one that produces un encrypted output.](encrypting-on-demand-smooth-streams/_static/image6.png) |
 
    Note the name of the watch folder that the job template is bound to in **Watch folders that use this job template**. We'll configure this watch folder to run the transformation jobs later in this article.
 3. In the **Actions** pane, click **Edit**.  
-   ![](encrypting-on-demand-smooth-streams/_static/image7.png)
+   ![Screenshot shows Actions pane to select Edit option.](encrypting-on-demand-smooth-streams/_static/image7.png)
```
Comment (RedZephyr13) [inline iis/media/transform-manager/encrypting-on-demand-smooth-streams.md:75]: How about "Screenshot that shows the Edit option on the Actions pane."

File: iis/media/transform-manager/encrypting-on-demand-smooth-streams.md [hunk 3]
```diff
@@ -63,29 +63,29 @@ You can also add the **PlayReady Protection** task to other built-in job templat
...
 4. In the **Edit Job Template** dialog box, on the **Basic Settings** tab, you can edit the job template **Name** and **Description** to give it more descriptive information for your environment.  
 
    | **Smooth Streams to encrypted Smooth Streams** | **Video files to H.264 Smooth Streams and Apple HTTP Live Streams** |
    | --- | --- |
-   | ![](encrypting-on-demand-smooth-streams/_static/image11.png) | ![](encrypting-on-demand-smooth-streams/_static/image12.png) |
+   | ![Screenshot shows Edit Job Template dialog box to edit name and description.](encrypting-on-demand-smooth-streams/_static/image11.png) | ![Screenshot shows the second job template to edit name and description fields.](encrypting-on-demand-smooth-streams/_static/image12.png) |
```
Comment (RedZephyr13) [inline iis/media/transform-manager/encrypting-on-demand-smooth-streams.md:80]: How about "Screenshot that shows the Edit Job Template dialog box for encrypting Smooth Streams."

File: iis/media/transform-manager/encrypting-on-demand-smooth-streams.md [hunk 4]
```diff
@@ -63,29 +63,29 @@ You can also add the **PlayReady Protection** task to other built-in job templat
...
 
-   The **Task definitions** list shows the tasks that are used by the job template.  
+   The **Task definitions** list shows the tasks that are used by the job template.
 
    > [!NOTE]
    > The **Edit Job Template** dialog box will display a task configuration error message on the **Messages** tab if a required property value for the **PlayReady Protection** task isn't specified. You can ignore this message for now as we'll configure the required task properties later in this article.
 
 5. If you selected the **Smooth Streams to encrypted Smooth Streams** job template, select the **PlayReady Protection** task in the **Task definitions** list, click **Edit**, and then [configure the **PlayReady Protection** task properties](encrypting-on-demand-smooth-streams.md#config_task).  
-   ![](encrypting-on-demand-smooth-streams/_static/image17.png)
+   ![Screenshot shows protection task in the task definitions list.](encrypting-on-demand-smooth-streams/_static/image17.png)
```
Comment (RedZephyr13) [inline iis/media/transform-manager/encrypting-on-demand-smooth-streams.md:88]: How about "Screenshots that show the Edit Job Template dialog box. Play Ready Protection is listed for the Task definitions and the Edit button is highlighted."

File: iis/media/transform-manager/encrypting-on-demand-smooth-streams.md (hunk 3 above)
Comment (v-lkasturi) [reply iis/media/transform-manager/encrypting-on-demand-smooth-streams.md:80]: I really like this! Thanks for your suggestion.

**Comment by v-lkasturi** [comment]
@Rick-Anderson @mcleblanc This PR fixes validation warnings/suggestions per a platform-wide push to increase content health scores. Please review at your convenience and sign off or merge if you approve or let me know if I need to make any other changes. CC: @MonicaRush @snoviking

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**